data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/config/GetDepositContract.java

```diff
@@ -38,16 +38,16 @@
   private final ConfigProvider configProvider;
 
   public GetDepositContract(
-      final Optional<Eth1Address> depositContractAddress,
+      final Eth1Address depositContractAddress,
       final JsonProvider jsonProvider,
       final ConfigProvider configProvider) {
     this.jsonProvider = jsonProvider;
     this.configProvider = configProvider;
     this.depositContractResponse = Optional.empty();
     this.depositContractAddress =
-        depositContractAddress
-            .map(Eth1Address::toHexString)
-            .orElse("0xdddddddddddddddddddddddddddddddddddddddd");
+        depositContractAddress == null
+            ? "0xdddddddddddddddddddddddddddddddddddddddd"
+            : depositContractAddress.toHexString();
```

Comment:
Do we still need this? It's very weird to be returning `0xdddddddddddddddddddddddddddddddddddddddd` and we should now always have a deposit contract as it's a required value in the network config file.

Reply:
only in the integration tests from memory. I can probably just investigate the tests and see why they broke but i figured this was equivalent to what was there already... thoughts?

Reply:
Would be nice to get rid of it, but don't spend too much time, it's not a big deal.